instrumentation/volley/library/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+import java.time.Duration
+
+plugins {
+    id("com.android.library")
+    id("otel.android-library-conventions")
+    id("otel.errorprone-conventions")
+}
+
+android {
+    namespace = "io.opentelemetry.android.volley"
+
+    compileSdk = 34
+    buildToolsVersion = "34.0.0"
+
+    defaultConfig {
+        minSdk = 21
```

Comment:
The values `minSdk` and `compileSdk` are defined in the conventions so it's better to leave them there only to try and keep consistency across all android libs in this repo (unless it's needed to override them depending on the case).

And regarding `buildToolsVersion`, I think it's safe to remove it since the AGP infers it from elsewhere (don't remember exactly where from though) but it's no longer mandatory to define it, which is nice IMO (a version less to take care of) 😌

Reply:
I bet this was just carry-over from the old way. Will fix. Thanks!